### ISSUE TITLE
[Windows] Error on using SMTP

### ISSUE THREAD

**Issue by danikarik** [laravel/framework#27110]
- Laravel Version: 5.6.39
- PHP Version: 7.2.13
- Database Driver & Version: mysql & 5.7.24

### Description:

I deployed Laravel on Windows Server 2016 and everything works fine, except mail driver. When trying to send email, I get error:

```
production.ERROR: Process could not be started [The system cannot find the path specified.
] {"exception":"[object] (Swift_TransportException(code: 0): Process could not be started [The system cannot find the path specified.
] at ...\\vendor\\swiftmailer\\swiftmailer\\lib\\classes\\Swift\\Transport\\StreamBuffer.php:299)
```

### Steps To Reproduce:

1. Create new Laravel app (no matter)
2. Configure `MAIL_DRIVER` to `SMTP`
3. `php artisan make:auth`
4. Deploy on **Windows Server 2016**
5. Submit Password Reset Form

**Comment by koenhoeijmakers** [comment]
As specified in your previous closed issue #27098, you should use the Issue Template properly.

**Comment by driesvints** [comment]
Heya, unfortunately we don't support this version anymore. Please check out [our support policy](https://laravel.com/docs/releases#support-policy) on which versions we are currently supporting. Can you please try to upgrade to the latest version and see if your problem persists? If so feel free to reply and we'll try to have a look.
